frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -264,7 +264,8 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
     k8sGet(InfrastructureModel, 'cluster')
       .then((infra: K8sResourceKind) => {
         // find infra supported provisioner
-        provisioner = infraProvisionerMap[_.lowerCase(getInfrastructurePlatform(infra))];
+        const infrastructure = getInfrastructurePlatform(infra);
+        provisioner = infraProvisionerMap[infrastructure.toLowerCase()];
```

Comment:
What's the context for moving this into separate lines. A description could help!
Thanks!